github/ci/prow/files/jobs/kubevirt/user-guide/user-guide-postsubmits.yaml

```diff
@@ -0,0 +1,47 @@
+postsubmits:
+  kubevirt/user-guide:
+  - name: push-user-guide-source-build-and-push-to-gh-pages
+    branches:
+    - ^source$
```

Comment:
The agreement from discussing this was to keep `master` for the source, so we should change this:
```suggestion
    - ^master$
```